integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -254,6 +264,68 @@ def export_unique_courses(self):
         ).order_by('course_id')
         return set(get_course_id_for_enrollment(enrollment) for enrollment in enrollment_queryset)
 
+    def get_grades_summary(
+        self,
+        course_details,
+        enterprise_enrollment,
+        channel_name,
+    ):
+        '''
+        Fetch grades info using either certificate api, or grades api.
+        Note: This logic is going to be refactored, so that audit enrollments are treated separately
+        '''
+        is_audit_enrollment = enterprise_enrollment.is_audit_enrollment
+        lms_user_id = enterprise_enrollment.enterprise_customer_user.user_id
+        enterprise_customer_uuid = enterprise_enrollment.enterprise_customer_user.enterprise_customer.uuid
+        course_id = enterprise_enrollment.course_id
+
+        # For instructor-paced and non-audit courses, let the certificate determine course completion
+        if course_details.pacing == 'instructor' and not is_audit_enrollment:
```

Comment:
to be removed in the future! But thank you for moving this to a helper function 🥳